src/web_asset_io.rs

```diff
@@ -68,19 +72,36 @@ impl AssetIo for WebAssetIo {
         self.default_io.read_directory(path)
     }
 
-    fn watch_path_for_changes(&self, path: &Path) -> Result<(), AssetIoError> {
-        if is_http(path) {
+    fn watch_path_for_changes(&self, _path: &Path) -> Result<(), AssetIoError> {
+        if is_http(_path) {
```

Comment:
Don't prefix arguments with _ unless they're unused

Reply:
This is directly copied from the FileAssetIo implementation; I thought this was weird, too, I don't know why I kept it in.